nav2_bringup/maps/map.yaml

```diff
@@ -1,6 +1,8 @@
-image: keepout_mask.pgm
+image: map.png
```

Comment:
Lets make this `warehouse.yaml` and `warehouse.png` to have a more specific name in case we add more maps in the future